src/index.js

```diff
@@ -5,12 +5,14 @@ const {
   parseURLEncodedText,
   parseURLParams,
   getSelectableInInspectorGrips,
+  maybeEscapePropertyName,
```

Comment:
We do, because there will be a corresponding patch to the object inspector, so that this works correctly in the debugger as well.